src/profile/helpers/editData.ts

```diff
@@ -105,15 +137,15 @@ function getValueProps(dataType: EditDataType): string[] {
   }
 }
 
-function pickValue(
+export function pickValue(
   profileDataItem: EditDataProfileSource,
   dataType: EditDataType
 ): EditDataValue {
   const pickProps = getValueProps(dataType);
   return _.pick(profileDataItem, pickProps) as EditDataValue;
 }
 
-function create(
+function createNewItem(
```

Comment:
The function definition (line 148) is pretty far away from where it is used first time in this file: line 303. I wonder if it would be possible to move it closer to usage and if it would increase readability?

Reply:
Referring to another re-ordering comment:
https://github.com/City-of-Helsinki/open-city-profile-ui/pull/154#discussion_r618158453